config/formats.js

```diff
@@ -479,1359 +479,16 @@ exports.Formats = [
-				pokemon.types = ["Bug", "Dragon"];
-			}
-			if (name === 'bullet' && !pokemon.illusion) {
-				this.add('-start', pokemon, 'typechange', 'Bug/Steel');
-				pokemon.types = ["Bug", "Steel"];
-			}
-			if (name === 'lasius' && !pokemon.illusion) {
-				this.add('-start', pokemon, 'typechange', 'Bug/Dark');
-				pokemon.types = ["Bug", "Dark"];
-			}
-			if (name === 'sweetant' && !pokemon.illusion) {
-				this.add('-start', pokemon, 'typechange', 'Bug/Fairy');
-				pokemon.types = ["Bug", "Fairy"];
-			}
-			if (name === 'geospirit' && !pokemon.illusion) {
-				this.add('-start', pokemon, 'typechange', 'Rock/Ghost');
-				pokemon.types = ["Rock", "Ghost"];
-			}
-			if (name === 'trex' && !pokemon.illusion) {
-				this.add('-start', pokemon, 'typechange', 'Rock/Dragon');
-				pokemon.types = ["Rock", "Dragon"];
-			}
-			if (name === 'tlizard' && !pokemon.illusion) {
-				this.add('-start', pokemon, 'typechange', 'Rock/Dark');
-				pokemon.types = ["Rock", "Dark"];
-			}
-			if (name === 'shield' && !pokemon.illusion) {
-				this.add('-start', pokemon, 'typechange', 'Rock/Steel');
-				pokemon.types = ["Rock", "Steel"];
-			}
-			if (name === 'pixelrock' && !pokemon.illusion) {
-				this.add('-start', pokemon, 'typechange', 'Rock/Fairy');
-				pokemon.types = ["Rock", "Fairy"];
-			}
-			if (name === 'draconic' && !pokemon.illusion) {
-				this.add('-start', pokemon, 'typechange', 'Ghost/Dragon');
-				pokemon.types = ["Ghost", "Dragon"];
-			}
-			if (name === 'souls' && !pokemon.illusion) {
-				this.add('-start', pokemon, 'typechange', 'Ghost/Dark');
-				pokemon.types = ["Ghost", "Dark"];
-			}
-			if (name === 'mask' && !pokemon.illusion) {
-				this.add('-start', pokemon, 'typechange', 'Ghost/Steel');
-				pokemon.types = ["Ghost", "Steel"];
-			}
-			if (name === 'fairydust' && !pokemon.illusion) {
-				this.add('-start', pokemon, 'typechange', 'Ghost/Fairy');
-				pokemon.types = ["Ghost", "Fairy"];
-			}
-			if (name === 'yamata' && !pokemon.illusion) {
-				this.add('-start', pokemon, 'typechange', 'Dragon/Dark');
-				pokemon.types = ["Dragon", "Dark"];
-			}
-			if (name === 'temporal' && !pokemon.illusion) {
-				this.add('-start', pokemon, 'typechange', 'Dragon/Steel');
-				pokemon.types = ["Dragon", "Steel"];
-			}
-			if (name === 'nicedragon' && !pokemon.illusion) {
-				this.add('-start', pokemon, 'typechange', 'Dragon/Fairy');
-				pokemon.types = ["Dragon", "Fairy"];
-			}
-			if (name === 'taintedmetal' && !pokemon.illusion) {
-				this.add('-start', pokemon, 'typechange', 'Dark/Steel');
-				pokemon.types = ["Dragon", "Steel"];
-			}
-			if (name === 'taintedfairy' && !pokemon.illusion) {
-				this.add('-start', pokemon, 'typechange', 'Dark/Fairy');
-				pokemon.types = ["Dark", "Fairy"];
-			}
-			if (name === 'pixieshield' && !pokemon.illusion) {
-				this.add('-start', pokemon, 'typechange', 'Steel/Fairy');
-				pokemon.types = ["Steel", "Fairy"];
-			}
-		},
+
+		mod: 'purebred',
+		ruleset: ['[Gen 7] OU'],
```

Comment:
iirc we want to avoid using other tiers as rulesets

Reply:
If its banlists for teambuilder tiers im ok with inheriting rulesets since all bans and unbans will happen automatically but ill leave that to @HoeenCoder to decide

Reply:
Its fine for teambuilder forrmats, not random formats.